src/sql-parser/src/parser.rs

```diff
@@ -3716,6 +3717,21 @@ impl<'a> Parser<'a> {
         }
     }
 
+    fn parse_kafka_include_header(&mut self) -> Result<SourceIncludeMetadata, ParserError> {
+        let key = self.parse_literal_string()?;
+        let alias = self
+            .parse_keyword(AS)
+            .then(|| self.parse_identifier())
+            .transpose()?;
```

Comment:
I think we should follow `CREATE SOURCE ... WEBHOOK` here in making the name mandatory. The reason is that headers are very likely to not be usable directly as column names (e.g., `mz-timestamp`), and so users will have to double quote them to use them (`"mz-timestamp"`), which is guaranteed to trip people out.